core/src/main/kotlin/Kord.kt

```diff
@@ -403,6 +405,13 @@ public class Kord(
             return KordRestOnlyBuilder(token).apply(builder).build()
         }
     }
+
+    public inline fun proxy(builder: KordProxyBuilder.() -> Unit = {}): Kord {
```

Comment:
This has to be in the companion object like `restOnly`.
Docs would be nice too.
Maybe `@KordExperimental` too?